frontend/src/hooks/useResourceAttribute/utils.ts

```diff
@@ -221,7 +220,9 @@ export const updateQuery = (
 
 export function getResourceAttributeQueriesFromURL(): IResourceAttribute[] {
 	const resourceAttributeQuery = new URLSearchParams(
-		history.location.search,
+		// history.location.search,
```

Comment:
I think, we can remove this with this update since we have access to searchParams in the new router version in the form of a hook